spec/app/tournament/tournament.spec.js

```diff
@@ -0,0 +1,121 @@
+import {Tournament} from "../../../app/tournament/tournament";
+import {Player} from "../../../app/tournament/player";
+
+describe('class Tournament', function() {
+
+    var spiderman = new Player("Spiderman", 1);
+    var batman = new Player("Batman", 3);
+    var theThing = new Player("The Thing", 5);
+
+    var tournament;
+
+    beforeEach( function() {
+        tournament = new Tournament();
+    });
+
+    function setMockedTournament(players) {
+        while (players--) {
+            tournament.addPlayer(spiderman);
+        }
+    }
+
+    it('should return false when searching for a non-enrolled player', function() {
+        expect( tournament.existsPlayer(spiderman) ).toBeUndefined();
```

Comment:
The point is I wrote false instead of undefined. *Nevertheless*, a method that asks "exists player?" should return, in my opinion, yes or no (true/false). It also lacks the spec where true is returned if player is found.